src/styles/_catalog.scss

```
@@ -1,10 +1,32 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 48px;
 
   max-width: 944px;
   margin: 0 auto;
 
+  @media screen and (max-width: 1023px) {
+    & {
+      grid-template-columns: repeat(3, 200px);
+      max-width: 696px;
+    }
+  }
```

```suggestion
  @media screen and (max-width: 1023px) {
      grid-template-columns: /*code*/;
  }
```
lets make it work like this everywhere